apps/astarte_appengine_api/lib/astarte_appengine_api/device/queries.ex

```diff
@@ -437,7 +437,7 @@ defmodule Astarte.AppEngine.API.Device.Queries do
         device_id,
         %InterfaceDescriptor{storage_type: :multi_interface_individual_datastream_dbtable} =
           interface_descriptor,
-        endpoint_id,
+        _endpoint_id,
```

Comment:
I think that we should fix any warning in release-0.11, just change code related to this PR.